.php_cs.dist

```diff
@@ -0,0 +1,61 @@
+<?php
+$finder = PhpCsFixer\Finder::create()
+    ->exclude([ 'Library' ])
+    ->in(__DIR__)
+;
+
+$header = <<<EOF
+(c) shopware AG <info@shopware.com>
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+EOF;
+
+return PhpCsFixer\Config::create()
+    ->setRules([
+        '@PSR2' => true,
+        'array_syntax' => ['syntax' => 'short'],
+        'blank_line_after_namespace' => true,
+        'blank_line_after_opening_tag' => true,
+        'blank_line_before_return' => true,
```

Comment:
I saw that in their repository. Maybe we use different version.
It's not critical, so we can replace at some point in the future.